content/en/docs/ambient/architecture/index.md

```diff
@@ -48,18 +98,18 @@ By default, both will be accepted by ztunnel.
 Because plaintext requests will have no peer identity when Authorization Policies are evaluated,
 a user can set a policy requiring an identity (either *any* identity, or a specific one) to block all plaintext traffic.
 
-When the destination is waypoint enabled, all requests *must* go through the waypoint where policy is enforced.
-The ztunnel will make sure this occurs.
-However, there is an edge case: a well behaving HBONE client (such as another ztunnel or Istio sidecar) would know to send to the waypoint, but other clients
-(such as a workload outside of the mesh) likely would not know anything about waypoint proxies and send requests directly.
-When these direct calls are made, the ztunnel will "hairpin" the request to its own waypoint to ensure policies are properly enforced.
+When the destination is waypoint enabled, if the source is `captured` by its ztunnel, the ztunnel ensures the request **must** go through
+the waypoint where policy is enforced. However, a workload outside of the mesh doesn't know anything about waypoint proxies therefore it sends
+requests directly to the destination without going through any waypoint proxy even if the destination is waypoint enabled.
+Currently, traffic from sidecars and gateways won't go through any waypoint proxy either and they will be made aware of waypoint proxies
+in a future release.
 
 ### Waypoint routing
 
 A waypoint exclusively receives HBONE requests.
 Upon receiving a request, the waypoint will ensure it is targeting either a `Pod` that it manages or a `Service` that contains a `Pod` it manages.
 
-For either type of request, the waypoint will enforce policies (such as `AuthorizationPolicy`, `WasmPlugin`, `Telemetry`, etc) before forwarding.
+For either type of request, the waypoint will enforce policies (such as `AuthorizationPolicy`, `RequestAuthentication`, `WasmPlugin`, `Telemetry`, etc) before forwarding.
```

Comment:
Not sure what is meant by 'type' of request. I think you covered this in the previous sentence so you could just skip. Alternatively can say something like

"Having accepted the traffic the waypoint will then enforce policies (such as `AuthorizationPolicy`, `RequestAuthentication`, `WasmPlugin`, `Telemetry`, etc) before forwarding.

Reply:
Updated